SlowCheetah.VisualStudio/SlowCheetahPackage.cs

```diff
@@ -63,6 +65,8 @@ public sealed class SlowCheetahPackage : Package
         public static SlowCheetahPackage OurPackage { get; set; }
```

Comment:
https://github.com/sayedihashimi/slow-cheetah/blob/master/SlowCheetah.VisualStudio/OptionsDialogPage.cs#L37
Among others. Need to verify if it is actually necessary